tests/lib/snaps/fuse-consumer/hello.py

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env python
+
+#    Copyright (C) 2006  Andrew Straw  <strawman@astraw.com>
+#
+#    This program can be distributed under the terms of the GNU LGPL.
+#    See the file COPYING.
+#
+
+import os, stat, errno
+# pull in some spaghetti to make this stuff work without fuse-py being installed
+try:
+    import _find_fuse_parts
```

Comment:
Let's please drop this module altogether, and do exactly what needs to be done here, if anything is necessary at all. And then, let's not ignore errors in that logic as done below.